### ISSUE TITLE
Update to Zabbix 5.2?

### ISSUE THREAD

**Issue by stevefxp1** [thompatry/zabbix-pihole#2]
Have you thought about updating this to incorporate support for Zabbix 5.2?

**Comment by thompatry** [comment]
Yep. I will get it brought up to 5.2 hopefully in the next week or so.

**Comment by stevefxp1** [comment]
Have you had a chance to update this to 5.4, now that this is out?

**Comment by thompatry** [comment]
I just tested it today and it still works. You will need to edit the shell script with the appropriate server instead of localhost. I will update the readme later today with the right information.

**Comment by lzm93** [comment]
Pode fazer para versão 6.4 um pouco mais detalhada por favor?
